public/css/drag-drop-style.css

```
@@ -0,0 +1,523 @@
+  transition:
+    visibility 0s 0.25s,
+    opacity 0.25s linear;
+}
+
+.answerContainer {
+  margin: 0 auto;
+  display: grid;
+  grid-template-columns: repeat(2, minmax(0, 200px));
+  justify-content: center;
+  justify-items: stretch;
+  grid-column-gap: 4%;
```

Stylelint (17.10.0)

[error] Expected "grid-column-gap" to be "column-gap" (property-no-deprecated)

(property-no-deprecated)